com.unity.toonshader/Editor/UTS3GUI.cs

```diff
@@ -13,7 +13,7 @@ internal partial class UTS3GUI : UnityEditor.ShaderGUI
 
         internal const float kVersionX = 0.0f;
         internal const float kVersionY = 9.0f;
-        internal const float kVersionZ = 5.0f;
+        internal const float kVersionZ = 6.0f;
 
         internal static string versionString => "0.9.5-preview";

```

Comment:
0.9.6-preview ?

Reply:
This is the repo for development and this commit is for the next version.  The version string should be gotton from the package information if we release the next one.